app/helpers/helpers.php

```diff
@@ -54,9 +54,6 @@ function sanitizeNumber($input) {
     }
     return null; 
 }
-function sanitizeTrim($input) {
-    return is_string($input) ? trim($input) : $input;
-}
 
```

Comment:
Why did you remove sanitizeTrim ?

Reply:
After pulling from develop, i got duplicate of that function. That function still exists in the helpers.